bclconvert.wdl

```diff
@@ -1,14 +1,10 @@
 version 1.0
 
 struct Sample {
-    Array[String] barcodes
+    Array[String]+ barcodes
     String name
 }
 
-struct SampleList {
-    Array[Sample]+ samples
-}
-
 struct FastqFile {
     String name
```

Comment:
I don't think name is required (as we're only supporting one sample) and removing it would simplify the output mapping, e.g. no need for sample name here: https://github.com/oicr-gsi/bclconvert/pull/1/files#diff-244b203ab17cb0b372f0bd5a657945a899d535bcb2bb5b3ea646ac424878a60dR66 - can it be removed please?

The output for this workflow should be `Array[Pair[File,Map[String,String]]]`

Reply:
sure

Reply:
Maybe that should be a separate workflow. I don't think this workflow has any intention of provision out undetermined reads correct?